pkg/server/commands/reverseexpand/reverse_expand.go

```diff
@@ -207,6 +210,11 @@ func (c *ReverseExpandQuery) Execute(
 	if err != nil {
 		return err
 	}
+	c.skippableErrorsMutex.RLock()
+	defer c.skippableErrorsMutex.RUnlock()
+	if c.skippableErrors.ErrorOrNil() != nil {
+		return c.skippableErrors.ErrorOrNil()
+	}
```

Comment:
i'm not too fond of this solution but it was the first approach that i could think of - i'm open to suggestions on how to make it better :)